### PR TITLE
Initial readme

### PR DESCRIPTION
**Pull request by jhanca-robotecai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
...
+
+## Folder structure
+
+Each asset is located in a separate folder, a subfolder of the Assets folder.
+
+In the raw-type gems, it is strongly advised to place all files used by the asset in its folder. Two or more assets may share the same files within one gem, e.g. materials and textures. In this case, these files will be located in the folder of one of these assets. It is advised not to use files from different gems, however, if this is necessary, such gem must be listed in the prerequisites and this should be mentioned in the gem's README file.
+
+Assets in the working-type gems may use assets from other gems with no limitations.
+
+Assets containing models (in raw-type gems) must be arranged in the following folder structure:
```
Comment (michalpelka) [inline README.md:61]:
```suggestion
It is strongly recommended that assets containing models (in raw-type gems) would be arranged in the following folder structure:
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,88 @@
...
+## Gems with "working" assets
+
+The purpose of these gems is to provide prefabs prepared to be used in different applications. These prefabs may be built around "raw" assets (hosted in raw-type gems) and should be extended by additional functionality. This may include for example:
+- Adding physics structure (rigid bodies, joints, etc.)
+- Adding ROS components
+- Adding scripts
+- Joining multiple prefabs (from raw-type or working-type gems)
+
+Working-type gems must not contain models, materials, or textures. Assets from raw-type gems should be used instead.
+
```
Comment (michalpelka) [inline README.md:44]: Working-type gem should not require any other gem other than :
- "Atom",
- "PrimitiveAssets",
- "AtomContent",
- "DiffuseProbeGrid",
- "DevTextures",
- "EMotionFX",
- "FastNoise",
- "GradientSignal",
- "Terrain",
- "Vegetation", 
- "WhiteBox"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,88 @@
...
+## Gems with "raw" assets
+
+The main purpose of these gems is to provide models, materials, and textures (including HDRi maps). If it is applicable, prefabs should be included integrating other objects of asset. The purpose of these prefabs is to facilitate inserting assets on the scene. Such prefabs however must not contain any functionality beyond the absolute minimum necessary to show assets correctly. Prefabs must not contain in particular:
+- Physics (rigid bodies, joints, etc.)
+- ROS components
+- Scripts
+
+Raw-type gems may depend only on official gems distributed with the O3DE engine. These dependencies should be reduced to a minimum. Raw-type gems must not depend on any external gems, including those contained in o3de-extras repository.
+
```
Comment (michalpelka) [inline README.md:34]:
```suggestion
Raw gems should also work on any version modern of O3DE (from 2310) and support Linux and Windows platform.
```

File: README.md (hunk 2 above)
Comment (pawelbudziszewski) [reply README.md:44]: IMO we shouldn't make such limitations. There are some other gems, I can imagine might be used (weather, clouds, vehicle dynamics, not to mention ROS2). And I believe, more will follow

**Comment by jhanca-robotecai** [comment]
The decision was made to redo the readme and keep it on a separate repository. I will close this PR for now and reopen when needed.

**Pull request closed without merging**